static/src/javascripts/projects/commercial/modules/dfp/Advert.js

```diff
@@ -16,7 +16,7 @@ type Resolver = (x: boolean) => void;
  * One size       - `data-mobile="300,50"`
  * Multiple sizes - `data-mobile="300,50|320,50"`
  */
-const createSizeMapping = (attr: string): Array<AdSize> =>
+const createSizeMapping = (attr: string): Array<SingleSize> =>
```

Comment:
Yeah, let's leave it aside for now. (besides I conflated an array type with an object type 😊 )

thanks for your advice!